eq-author/src/App/QuestionnaireDesignPage/index.js

```diff
@@ -73,6 +73,10 @@ export const QuestionnaireDesignPage = () => {
     data: { questionnaire } = {},
   } = useQuestionnaireQuery(questionnaireId);
 
+  const themeErrorCount = questionnaire?.themeSettings?.validationErrorInfo?.errors.filter(
```

Comment:
```suggestion
  const formTypeErrors = questionnaire?.themeSettings?.validationErrorInfo?.errors.filter(
```

Could this be renamed to make it a little clearer that it's just form type errors, rather than all theme errors, please?